.expeditor/buildkite/deploy_docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eou pipefail
+
+aws-configure chef-cd
+
+ceval "aws s3 sync build s3://chef-web-docs-${acceptance}.cd.chef.co --profile chef-cd"
```

Comment:
you aren't setting the variable `acceptance` - did you mean for this to just be a full string?

Reply:
Copy paste error - should have been ENVIRONMENT